snakemake/remote/zenodo.py

```diff
@@ -0,0 +1,212 @@
+__author__ = "Taavi Päll"
+__copyright__ = "Copyright 2019, Taavi Päll"
+__email__ = "tapa741@gmail.com"
+__license__ = "MIT"
+
+import os
+import re
+import hashlib
+from collections import namedtuple
+
+from snakemake.remote import AbstractRemoteObject, AbstractRemoteProvider
+from snakemake.exceptions import ZenodoFileException, WorkflowError
+from snakemake.common import lazy_property
+from snakemake.utils import makedirs
+
+try:
+    # Third-party modules
+    import requests
+    from requests.exceptions import HTTPError
+except ImportError as e:
```

Comment:
Exception handling is not needed here, requests is already a default dependency of snakemake.

Reply:
Removed exception. Whereas, I just added it based on other remote provider examples:-)

Reply:
Thanks! Those might be historical.